libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Actions/SwitchCondition.cs

```diff
@@ -38,7 +38,7 @@ public SwitchCondition([CallerFilePath] string callerPath = "", [CallerLineNumbe
         /// Value Expression against memory. This value expression will be combined with value expression in case statements to make a bool expression.
         /// </value>
         [JsonProperty("condition")]
-        public ValueExpression Condition { get; set; } 
+        public Expression Condition { get; set; }
```

Comment:
>Expression [](start = 15, length = 10)

You want to require an expression?

Reply:
yes

Reply:
Yes, it doesn't make any sense to say 
switch (13)
or 
switch("Hello")